spec.html

```diff
@@ -24069,6 +24069,34 @@ <h1>Object.defineProperty ( _O_, _P_, _Attributes_ )</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-Object.fromEntries">
```

Comment:
To match the rest of the spec, I think this should be spelled `sec-object.fromentries`. Existing sections are


```
      <emu-clause id="sec-object.assign">
      <emu-clause id="sec-object.create">
      <emu-clause id="sec-object.defineproperties">
      <emu-clause id="sec-object.defineproperty">
      <emu-clause id="sec-object.entries">
      <emu-clause id="sec-object.freeze">
      <emu-clause id="sec-object.getownpropertydescriptor">
      <emu-clause id="sec-object.getownpropertydescriptors">
      <emu-clause id="sec-object.getownpropertynames">
      <emu-clause id="sec-object.getownpropertysymbols">
      <emu-clause id="sec-object.getprototypeof">
      <emu-clause id="sec-object.is">
      <emu-clause id="sec-object.isextensible">
      <emu-clause id="sec-object.isfrozen">
      <emu-clause id="sec-object.issealed">
      <emu-clause id="sec-object.keys">
      <emu-clause id="sec-object.preventextensions">
      <emu-clause id="sec-object.prototype">
      <emu-clause id="sec-object.seal">
      <emu-clause id="sec-object.setprototypeof">
      <emu-clause id="sec-object.values">
      <emu-clause id="sec-object.prototype.constructor">
      <emu-clause id="sec-object.prototype.hasownproperty">
      <emu-clause id="sec-object.prototype.isprototypeof">
      <emu-clause id="sec-object.prototype.propertyisenumerable">
      <emu-clause id="sec-object.prototype.tolocalestring">
      <emu-clause id="sec-object.prototype.tostring">
      <emu-clause id="sec-object.prototype.valueof">
```